index.js

```diff
@@ -91,8 +91,8 @@ function getFlags(cb) {
 
     var index = result.indexOf('\nOptions:');
     if (index >= 0) {
+      result = result.slice(index);
       var regexp = /^\s\s--[\w-]+/gm;
```

Comment:
Does adding the `y` flag work?

```
var regexp = /^\s\s--[\w-]+/gmy;
regexp.lastIndex = index;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/sticky#description

Reply:
MDN says that `String#match` returns an array of all exec() results for `g` or `gy` flag, but actually does not return expected results.
```
let str = '  --xxx   --yyy   --zzz';
let regexp = /^\s\s--[\w-]+/g;                                                  
console.log(`str.match(${regexp}) = ${str.match(regexp)}`);
regexp = /^\s\s--[\w-]+/gy;
console.log(`str.match(${regexp}) = ${str.match(regexp)} (regexp.lastIndex = ${regexp.lastIndex})`);
regexp.lastIndex = 7;
console.log(`str.match(${regexp}) = ${str.match(regexp)} (regexp.lastIndex = ${regexp.lastIndex})`);

str = '  --xxx\n  --yyy\n  --zzz\n';
regexp = /^\s\s--[\w-]+/gm;
console.log(`str.match(${regexp}) = ${str.match(regexp)}`);
regexp = /^\s\s--[\w-]+/gmy;
console.log(`str.match(${regexp}) = ${str.match(regexp)} (regexp.lastIndex = ${regexp.lastIndex})`);
regexp.lastIndex = 7;
console.log(`str.match(${regexp}) = ${str.match(regexp)} (regexp.lastIndex = ${regexp.lastIndex})`);
--
% node regexp_stick_test.js
str.match(/^\s\s--[\w-]+/g) =   --xxx
str.match(/^\s\s--[\w-]+/gy) =   --xxx (regexp.lastIndex = 0)
str.match(/^\s\s--[\w-]+/gy) =   --xxx (regexp.lastIndex = 0)
str.match(/^\s\s--[\w-]+/gm) =   --xxx,  --yyy,  --zzz
str.match(/^\s\s--[\w-]+/gmy) =   --xxx (regexp.lastIndex = 0)
str.match(/^\s\s--[\w-]+/gmy) =   --xxx (regexp.lastIndex = 0)
%
```

In addition, `y` flag cause an eslint error because it is supported in Node.js `6.0.0` and later but this package (and other gulp packages) uses [eslint-config-gulp](https://github.com/gulpjs/eslint-config-gulp) which still supports syntax of Node.js `>= 0.10`.

Reply:
Although this package supports Node.js 10+, not 0.10+ 😅  It even uses features introduced in Node.js 10, such as `process.allowedNodeEnvironmentFlags`.

https://github.com/gulpjs/v8flags/blob/1f70fd1ce962dbf426a5fd17c7637248fc71c200/package.json#L16

Reply:
Our lint rules are in place to avoid certain community members from wasting our time as we work to release gulp 5. Once it is released, all linting and projects will be updated to support node 20+